search-services/latest/using/index.md

```diff
@@ -117,7 +117,7 @@ To search for an exact term you must prefix it with "=". The supported syntax:
 
 If you don’t specify a field the search runs against name, description, title, and content. If the field specified is `TOKENIZED=false`, only the full field is matched. If the field you specified is `TOKENIZED=TRUE` or `TOKENIZED=BOTH` then the search is run on the cross locale tokenized version of the field.
 
-> **Note:** If cross locale is not configured for the field then an exception occurs.
+> **Note:** If cross locale is not configured for the field then a 501 status code will be returned.
 
```

Comment:
Not sure this is the right place to add this 501 status, as the code is only returned from Alfresco REST API. May we add some clarification on that?

Reply:
Ok thanks - rephrased to make this clearer.